src/chrome/content/rules/NYDailyNews.xml

```diff
@@ -70,29 +72,27 @@ Fetch error: http://nydailynews.com/ => http://nydailynews.com/: (60, 'SSL certi
 			- i.simpli.fi *
 
 	* Secured by us
-
 -->
+
 <ruleset name="NYDailyNews (partial)">
 
-	<target host="nydailynews.com" />
-	<target host="*.nydailynews.com" />
-		<!--
-			Avoid false/broken MCB:
-						-->
-		<exclusion pattern="^http://classifieds\.nydailynews\.com/+(?!resource/)"/>
-		<exclusion pattern="^http://www\.nydailynews\.com/+(?!cmlink/|nydn/|polopoly_fs/)" />
+	<target host="beta.nydailynews.com" />
+	<target host="m.nydailynews.com" />
+	<target host="myaccount2.nydailynews.com" />
+	<target host="placeanad.nydailynews.com" />
+	<target host="static1.nydailynews.com" />
+	<target host="static2.nydailynews.com" />
+	<target host="static3.nydailynews.com" />
+	<target host="webport.nydailynews.com" />
 
+	<!-- Used on classifieds.nydailynews.com -->
+	<target host="dmpwow64jb5ov.cloudfront.net" />
 
 	<securecookie host="^[^@:/]*\.nydailynews\.com$" name=".+" />
 
-
-	<rule from="^http://(?:static\d|www)\.nydailynews\.com/"
+	<rule from="^http://static\d\.nydailynews\.com/"
```

Comment:
Please add some test urls for this.

Reply:
I actually can't find any usage of `static[1-3].nydailynews.com` anymore, so going to remove the rule instead.